actors/miner/src/lib.rs

```diff
@@ -5144,6 +5185,35 @@ fn verify_aggregate_seal(
     .context_code(ExitCode::USR_ILLEGAL_ARGUMENT, "aggregate seal verify failed")
 }
 
+fn validate_sealer_id_numbers(
+    rt: &impl Runtime,
+    sealer_id_actor: ActorID,
+    sealer_id_verifier_signature: Vec<u8>,
+    sector_numbers: &BitField,
```

Comment:
small nit: its mildly less confusing to call this sealer_numbers here and in the ActivateSectorParams